pyuvsim/analyticbeam.py

```diff
@@ -64,22 +85,26 @@ def interp(self, az_array, za_array, freq_array, reuse_spline=None):
             interp_data[0, 0, 0, :, :] = 1
             interp_basis_vector = None
         elif self.type == 'gaussian':
-            if self.sigma is None:
-                raise ValueError("Sigma needed for gaussian beam -- units: radians")
+            if (self.diameter is None) and (self.sigma is None):
+                raise ValueError("Dish diameter needed for gaussian beam -- units: meters")
             interp_data = np.zeros((2, 1, 2, freq_array.size, az_array.size), dtype=np.float)
             # gaussian beam only depends on Zenith Angle (symmetric is azimuth)
-            # standard deviation of sigma is refereing to the standard deviation of e-field beam!
-            values = np.exp(-(za_array**2) / (2 * self.sigma**2))
```

Comment:
The Gaussian here is in units of zenith-angle rather than angle cosines. Since angle cosines are often employed in the Gaussian approximation of an airy beam I would make sure to be explicit about the choice of angle units and consider adding an option for a gaussian of angle cosines.

Reply:
All of our primary beam models are evaluated at angular positions, no angle cosines, as indicated by the labeling of `za_array` and `az_array`. Throughout pyuvsim, there's no point where a direction cosine is used, and since we're not taking any flat sky approximations I can't think of a case where we'd need one.

I'll add a bit to the documentation to confirm that the  that the gaussian beam "sigma" parameter is in zenith angle. I will also make it more clear in the docstring for `diameter_to_sigma`, since sigma is